### PR TITLE
Tests: correct test types. Tests for wallet addition 

### PR DESCRIPTION
**Pull request by NotEternal**
## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/swaponline/MultiCurrencyWallet/blob/master/docs/CONTRIBUTING.md) guide
- [x] Good naming (as clear and simple as possible)
- [ ] Correct behavior if external API endpoints are down, return 404, 504 (or no answer), 401 errors (ddos simulation)
- [ ] I tested desktop/mobile resolution
- [ ] I tested light/dark theme
- [ ] I tested different languages
- [ ] I checked the functionality once again (**AFFECT MONEY**)
- [ ] I checked work on the Testnet
- [ ] I checked work on the Mainnet
- [ ] I checked the **PR** once again

Related to #4575

### CODE REVIEW

**Comment by NotEternal** [comment]
Test isn't working because of this issue #4137

File: src/front/shared/redux/actions/btc.ts [hunk 1]
```diff
@@ -442,8 +442,16 @@ const addressIsCorrect = (address) => {
...
-const send = ({ from, to, amount, feeValue = null, speed,  serviceFee = hasAdminFee }) => {
-  console.log('>>> send', from, to, amount, feeValue, speed)
+const send = (params) => {
+  const {
+    from,
+    to,
+    amount,
+    feeValue,
+    speed, 
+    serviceFee = hasAdminFee,
```
Comment (VitaliyShulik) [inline src/front/shared/redux/actions/btc.ts:452]: Вроде `serviceFee: hasAdminFee` должно быть

File: tests/e2e/utils.ts [hunk 2]
```diff
@@ -38,17 +42,23 @@ type ImportWalletParams = {
...
 
   await page.waitForSelector('.react-tags__search-input', {
     timeout,
   })
 
-  const wordInput = await page.$(`.react-tags__search-input`)
+  const wordInput: puppeteer.ElementHandle | null = await page.$(`.react-tags__search-input`)
+
+  if (!wordInput) {
+    throw new Error('HTML input for for the Mnemonic phrase is not found')
```
Comment (VitaliyShulik) [inline tests/e2e/utils.ts:57]: remove second `for`

File: tests/e2e/wallet/walletAddition.test.ts [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    ['matic', 'MATIC Token'],
+  ]
+
+  async function checkWalletDisplay(params) {
+    const { page, name, expectedTitle } = params
+
+    const walletTitle = await page.$eval(`#${name}WalletTitle`, el => el.textContent)
+
+    if (walletTitle !== expectedTitle) {
+      throw new Error('incorrect display for ${}')
```
Comment (VitaliyShulik) [inline tests/e2e/wallet/walletAddition.test.ts:27]: зачем `${}`?

File: src/front/shared/redux/actions/btc.ts (hunk 1 above)
Comment (NotEternal) [reply src/front/shared/redux/actions/btc.ts:452]: не, двоеточие для типов, равно для дефолтных параметров. Ну или если это JS, то двоеточие для нового имени, но сдесь именно параметр с самого начала файла

**Review by VitaliyShulik — APPROVED** [review]

**Pull request merged**